cocotb/share/makefiles/Makefile.pylib.Msys

```diff
@@ -29,18 +29,18 @@
 
 # All common python related rules
 
+PYTHON_BIN?=python.exe
+
 # if not explicitly set, auto-detect python dir from system path
 ifeq ($(PYTHON_DIR),)
-	PYTHON_DIR ?= $(shell dirname $(shell which python))
+    PYTHON_DIR ?= $(shell dirname $(shell which $(PYTHON_BIN)))
 endif
 
 # make sure python dir was set properly
 ifeq ($(PYTHON_DIR),)
-	$(error "Path to Python directory must be included in system path or defined in PYTHON_DIR environment variable")
+    $(error "Path to Python directory must be included in system path or defined in PYTHON_DIR environment variable")
 endif
```

Comment:
I think this `if` can be removed entirely, and the `ifeq` above should set `PYTHON_DIR` unconditionally.